src/components/reviews/ReviewsList.jsx

```diff
@@ -8,57 +9,80 @@ import SortReviews from './SortReviews.jsx';
 const ReviewsListContainer = styled.div`
   display: flex;
   flex-direction: column;
+  overflow: scroll;
```

Comment:
Can this be changed to `overflow-y: scroll;`? That way only a vertical scroll bar pops up.